src/bootstrap.php

```diff
@@ -65,6 +66,10 @@
         'profiler.cache_dir' => __DIR__.'/../var/cache/profiler',
         'profiler.mount_prefix' => '/_profiler',
     ]);
+    $app->get('/error', function () use ($app) {
+        $app['logger']->error('Simulating error');
+        throw new LogicException('Simulated error');
+    });
```

Comment:
Suggest we have a high priority here. The error handler https://github.com/elifesciences/api-problem-php/blob/master/src/Silex/ApiProblemProvider.php#L22-L26 has a low priority, but to avoid future problems we shouldn't rely on the default.

Reply:
(Comment should have been on the `$app->error()` of course.)